PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/map/TeamIntroPresentation.java

```diff
@@ -187,9 +188,9 @@ public static GroupZoom setTargets(IContest contest, String groupId, int height,
 					minLon = Math.min(minLon, lon);
 					maxLon = Math.max(maxLon, lon);
 				}
-				String label = t.getId() + " - " + t.getActualDisplayName();
+				String label = t.getLabel() + " - " + t.getActualDisplayName();
 				if (showOrganizations) {
-					label = org.getFormalName();
+					label = t.getLabel() + " - " + org.getFormalName();
```

Comment:
I think this is not what we want? If `showOrganizations` is set, we don't want team info do we?

Reply:
I waffled when doing the change, but you're right. Removed this and merging.